src/api.js

```diff
@@ -1,9 +1,13 @@
 import axios from "axios";
 
-export const getArticles = () => {
-  const response = axios
-    .get("https://articles-service-api.onrender.com/api/articles")
-    .then((articles) => articles.data);
+export const getArticles = (topicName) => {
+  let url = `https://articles-service-api.onrender.com/api/articles`;
```

Comment:
Great that you're using axios, but you could make better use of the axios functionality (specifically the params and baseURL) to do a lot of this logic for you https://notes.northcoders.com/courses/js-front-end/axios#axios-params